packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,30 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// The name of the printable key, encoded as UTF-8. This is
+  /// typically the character that key would produce without any modifier keys.
+  /// For dead keys, it is typically the diacritic it would add to a character.
```

Comment:
This could also specify the default value, and that it cannot be null since it is asserted.